sdk/python/feast/repo_config.py

```diff
@@ -91,6 +102,10 @@ def get_registry_config(self):
         if isinstance(self.registry, str):
             return RegistryConfig(path=self.registry)
         else:
+            if self.registry.path and self.registry.path[-1] != "/":
+                raise Exception(
+                    f"Path {self.registry.path} is not a valid directory path; please end with '/'."
+                )
```

Comment:
Note: going to enforce this because otherwise can't tell if it's a filepath or dirpath (can't distinguish between the old simple string 2 lines above `RegistryConfig(path=self.registry)`) - should be dirpath if using the larger config with option to publish json.